File: Front/Library/Filters/UpdateContactFilter.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Library.Filters
{
    public class UpdateContactFilter
    {
        public string ContactName { get; set; }

        public string ContactSurname { get; set; }

        public string ContactPhoneNumber { get; set; }

        public long ContactId { get; set; }
    }
}

File: 2_8/src/main/java/com/aleks/_8/DAL/Filters/UpdateContactFilter.java
package com.aleks._8.DAL.Filters;

import lombok.AllArgsConstructor;

@AllArgsConstructor
public class UpdateContactFilter {
    private String ContactName;

    public String getContactName() {
        return ContactName;
    }

    private String ContactSurname;

    public String getContactSurname() {
        return ContactSurname;
    }

    private String ContactPhoneNumber;

    public String getContactPhoneNumber() {
        return ContactPhoneNumber;
    }

    private int ContactId;

    public int getContactId() {
        return ContactId;
    }
}
